boot/zephyr/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+# Ensure that the qspi driver is disabled by default
+CONFIG_NORDIC_QSPI_NOR=n
+
+# TODO: below are not yet supported and need fixing
+CONFIG_NRF_RTC_TIMER=n
```

Comment:
Not sure why `CONFIG_NRF_RTC_TIMER` is being touched here, if I remember the PR in zephyr correctly then this Kconfig has been changed to only be selectable if grtc is not selected and grtc should be selected by default?